src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import NavBarItem from './NavBarItem';
+import './nav.css';
+
+class NavBar extends Component {
+    
+	state = { 
```

Comment:
Can we keep indexing consistent by using spaces instead of tabs?  Also keep indexing to 2 spaces for javascript files.

Reply:
Not sure why some of these files came in with 4 spaces... will resolve this